test/verify/check-blueprint

```diff
@@ -0,0 +1,142 @@
+#!/usr/bin/python3
+
+# checkpoint:
+# 1. create blueprint
+# 2. blueprint filter
+# 3. blueprint sort
+
+
+import parent
+import testlib
+
+
+@testlib.nondestructive
+class TestBlueprint(testlib.MachineCase):
+    def setUp(self):
+
+        super().setUp()
+        for message in parent.allowed_journal_messages:
+            self.allow_journal_messages(message)
```

Comment:
it takes n arguments for patterns, so you can just unpack the array and don't need to iterate over it :)

Reply:
Thanks. Updated!